dtfs-central-api/src/services/tfm/deal-cancellation.service.submitDealCancellation.test.ts

```diff
@@ -274,12 +274,13 @@ describe('DealCancellationService', () => {
           expect(dealCancellationResponse).toEqual(DealCancellationService.getTfmDealCancellationResponse(mockRepositoryResponse));
         });
 
-        it('does not call PortalDealService.updateStatus', async () => {
+        it(`it calls PortalDealService.updateStatus with ${DEAL_STATUS.PENDING_CANCELLATION} status`, async () => {
```

Comment:
Can we/do we have a test for when `PortalDealService.updateStatus` errors?

Reply:
There's no logic in the service to catch the errors - I can add it but I dont think its necessary in the way we want to check controllers convert `ApiErrors` into a status and a body

Reply:
Thanks @MarRobSoftwire - had a look and understand now there's no e.g try/catch handling here and the error will bubble up to a controller 👍